### ISSUE TITLE
[Bug]: UnableToRetrieveMetadata Runtime Exception on uploading files with certain names

### ISSUE THREAD

**Issue by esfredderickmx** [tasteui/tasteui#543]
### What Happened

Is not very known for the community that Livewire **can't handle file uploads on files with certain names**. This should be considered by the `<x-upload/>` component and throw an unexpected error message like "_Please rename the file or try to upload a different one_.".

### Component

Upload

### How to Reproduce

Make a fresh instance of a project using Laravel and TallstackUI. Add the `<x-upload/>` component and try to upload a file with a name such as "BdXjH2BDBgxpKPOy-fdebl7CVW2EDGy6DGWkH_MzjLTArxACXmvMxVeeoRathFFAKlfrWY-MF8Htnt8DdfVUq6Pqwch96cVO1YUKe0LqtiDZ-lRF8asDw4Xf9udWDaWKwMnMV3aV0Q6fBgP_c23IQnggY-37dxiabJdZRtOD3oezo6tIh9BlSTmgf04l6g6oLqFVr5J4cZdZE6IMQRD4.png". Then you get the next error message:
![image](https://github.com/tallstackui/tallstackui/assets/136201412/7e488757-d10e-4684-8dec-95edd4cd84ce)

### Laravel Version

11.14

### Livewire Version

3.5.2

### TallStackUI Version

1.32.0

### PHP Version

8.3.7

### Operation System

Windows

**Comment by devajmeireles** [comment]
Thank you very much for your report!

Although the solution you proposed seems to be the easiest, throwing yet another exception would not actually "solve the problem". With this, I thought it prudent to just "protect" obtaining the file size - which is what generates the exception and leave the responsibility for validating the file to the application, using Laravel's basic validations for PDF files:

```php
#[Validate(['required', 'mimes:pdf', 'mimetypes:application/pdf'])]
public $file = null;
```

Therefore, I prefer to believe that the solution provided here will be useful in solving this problem: https://github.com/tallstackui/tallstackui/pull/545, since it is not TallStackUI's responsibility to validate or ask the end user to rename a file.

Released now in v1.32.1. Thanks again!

**Comment by esfredderickmx** [comment]
It works like a charm! Thanks so much for answer this issue solving it asap! Now I can at least catch the exception and throw an error message :)
